PrimeNumberNaive/PrimeNumberNaive.java

```diff
@@ -0,0 +1,21 @@
+/*
+program calculate of prime number (2...n) when 'n' is input from user
+*/
+
+import java.util.Scanner;
+
+public class PrimeNumberNaive {
+	
+	public static void main(String[] args) {
+	
+	System.out.print("please enter of 'n' number of range (2...n): ");
```

Comment:
will work with 1e+20? if not, than what's the largest number that there is avilble?

Reply:
it works with 1e+9 max. I will change on long type (9e+18)